tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/TaskController.java

```diff
@@ -230,6 +235,13 @@ private void checkTaskImplementation(final LazySupplier<TaskDTO> taskSupplier) {
   }
 
   private boolean hasAccessToTask(final LazySupplier<TaskDTO> taskSupplier) {
+    if (userReader.getCurrentUser().getUserId() == null
+        || userReader.getCurrentUser().getUserId().isEmpty()) {
```

Comment:
we can invoke `userReader.getCurrentUser()` only once (it is not just a getter :) )
```suggestion
    if (StringUtils.isEmpty(userReader.getCurrentUser().getUserId())) {
```
same in `searchTasks`

Reply:
Ah makes sense, I've now called it once instead